test/protocol_test.py

```diff
@@ -1377,6 +1377,22 @@ def test_droplet_size(self, dummy_protocol):
                 "1.31:microliter",
             )
 
+    def test_propagate_properties(self, dummy_protocol):
+        p = dummy_protocol
+        p.propagate_properties = True
+        echo = p.ref("echo", None, "384-echo", discard=True)
+        dest = p.ref("dest", None, "384-echo", discard=True)
+        echo.well(0).set_properties({"test_well_property": True})
+        dest.well(0).set_properties({"test_well_property": False})
+        p.acoustic_transfer(
+            echo.well(0).set_volume("2:microliter"), dest.wells(0, 1), "50:nanoliter"
+        )
```

Comment:
super nit, but can you call `echo.well(0).set_volume("2:microliter")` above this input? then have this source arg just be `echo.well(0)` 

i think it'd be more readable, ie

```suggestion
        dest.well(0).set_properties({"test_well_property": False})
        echo.well(0).set_volume("2:microliter")

        p.acoustic_transfer(
            echo.well(0), dest.wells(0, 1), "50:nanoliter"
        )
```